extension/src/popups/MinervaMenu.html

```diff
@@ -16,72 +16,102 @@
   <body>
     <div id="extensionBody">
       <div class="collapse" id="minervaMenuCollapseCard">
-        <div class="card card-body">
-          <ul class="nav nav-pills justify-content-center">
-            <li class="nav-item">
-              <a class="nav-link active" id="you-tab" data-toggle="tab" href="#nav-you" role="tab" aria-controls="you"
-                aria-selected="true">You</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" id="ranking-tab" data-toggle="tab" href="#nav-ranking" role="tab" aria-controls="ranking"
-                aria-selected="false">Ranking</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" id="store-tab" data-toggle="tab" href="#nav-store" role="tab" aria-controls="store"
-                aria-selected="false">Store</a>
-            </li>
-          </ul>
-          <div class="tab-content" id="nav-tabContent">
-            <div class="tab-pane fade show active overflow-auto" id="nav-you" role="tabpanel" aria-labelledby="nav-you-tab">
-              <div class="card">
-                <div class="card-body">
-                  <h5 class="card-title" id="studentName-youPanel" ></h5>
-                  <p class="card-text">Total minutes watched: 69:42</p>
+        <div class="container">
```

Comment:
looks like a container effect, would this be the reason where I am seeing a tight box around various stats inside `You` tab? Otherwise, I don't think we this container here

Reply:
This container is used to separate the row of stickers from the rest of the student statistics.
The container is required for the "row" and "col" classes to properly separate.